CenterForOpenScience/ember-osf-web
# [ENG-4522] Cesium/no global css part 1 / 2

-   Ticket: [ENG-4522]
-   Feature flag: n/a

## Purpose

Remove global css everywhere except the app hierarchy. With some exceptions, like form control, which have interactions between components that cannot be easily localized. In those cases, add the global css to the _global.scss file.

## Summary of Changes

See commit list

## Side Effects

Too many potential side effects to count, all css related.

## QA Notes

This will probably take a general regression along with the removal of bootstrap itself. Be sure to check the scheduled banner image responsiveness.

[ENG-4522]: https://openscience.atlassian.net/browse/ENG-4522?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
